index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html id="main">
+    <head>
+        <meta charset="UTF-8">
+        <title>web witch</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Varela&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="file:///Users/chan/Developer/projects/portfolio/styles.css"> 
+    </head>
+    
+    <body>
+        <header>
+            <h1>hello.</h1>
+        </header>
+
+        <main>
+            <p class="intro">i'm chan. i really like building things.</p>
+
+        </main>
+    
+
+        <nav>
+            <ul id="nav_shapes">
+                <li id="transform"><a class= "shape1" href="about.html"><img src="file:///Users/chan/Developer/projects/personal-portfolio-site/images/about-me.png" alt="about me"></a></li>
```

Comment:
See comment about **relative paths**

In this example, the path to get to your `about-me.png` is `./images/about-me.png`. If we try to verbalize the path it reads like:

> "From the project root `.` in the images folder `/images` get the about-me image `/about-me.png`"

```suggestion
                <li id="transform"><a class= "shape1" href="about.html"><img src="./images/about-me.png" alt="about me"></a></li>
```